packages/main/src/tray-animate-icon.spec.ts

```
@@ -32,6 +32,23 @@ class TestAnimatedTray extends AnimatedTray {
   override isProd(): boolean {
     return super.isProd();
   }
+
+  override createTrayImage(iconName: string): Electron.NativeImage {
+    return super.createTrayImage(iconName);
+  }
+
+  override animateTrayIcon(): void {
+    return super.animateTrayIcon();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)